src/v2/spec.yaml

```diff
@@ -4356,6 +4390,117 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/WebhookListResponse"
+  /farcaster/mute/list:
+    get:
+      tags:
+        - Mute
+      summary: Get fids that a user has muted.
+      description: Fetches all fids that a user has muted.
+      operationId: mute-list
+      parameters:
+        - $ref: "#/components/parameters/ApiKey"
+        - name: fid
+          in: query
+          required: true
+          schema:
+            $ref: "#/components/schemas/Fid"
+          description: The user's fid (identifier)
+          example: 194
+        - name: limit
+          in: query
+          description: Number of results to retrieve (default 20, max 100).
+          required: false
+          schema:
+            type: integer
+            default: 20
+            minimum: 1
+            maximum: 100
+          example: 20
+        - name: cursor
+          in: query
+          description: Pagination cursor.
+          required: false
+          schema:
+            type: string
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/MuteListResponse"
+        "400":
+          $ref: "#/components/responses/400Response"
+        "404":
+          $ref: "#/components/responses/400Response"
+  /farcaster/mute:
+    post:
+      tags:
+        - Mute
+      summary: Adds a mute for a fid.
+      description: Adds a mute for a given fid.
```

Comment:
@MrKevinOConnell can we add a line here saying that this API is available to whitelisted customers only during testing period and they should reach out if they want to test?